src/components/footer.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import '../fontawesome/css/all.min.css';
+import '../fontawesome/css/fontawesome.min.css';
+
+const Footer = () => (
+  <li className='footer-wrapper'>
+    <div className='footer-logo'>
+      <a href="http://mikstura.it" className='footer-link'>mikstura<span className="header-link__it">.IT</span></a>
+    </div>
+    <ol className='footer__icon'>
+      <a href="https://www.meetup.com/Mikstura-IT/" className='footer-link' activeClassName='active'><i class="fab fa-meetup"></i></a>
```

Comment:
I guess `activeClassNames` it not necessary in this cases (in all social links) ✂️